shared/src/commonTest/kotlin/org/hyperskill/study_plan/widget/StudyPlanWidgetTest.kt

```diff
@@ -258,11 +258,43 @@ class StudyPlanWidgetTest {
         val (state, _) =
             reducer.reduce(
                 initialState,
-                StudyPlanWidgetFeature.LearningActivitiesFetchResult.Success(sectionId = sectionId, emptyList())
+                StudyPlanWidgetFeature.LearningActivitiesFetchResult.Success(
+                    sectionId = sectionId,
+                    activities = listOf(stubLearningActivity(1L))
+                )
             )
         assertEquals(StudyPlanWidgetFeature.ContentStatus.LOADED, state.studyPlanSections[sectionId]?.contentStatus)
     }
 
+    @Test
+    fun `Current section should be removed if no available activities loaded and next section should be expanded`() {
```

Comment:
Let's also write a test for the not current section. I'm sure that our code now works correctly, just want to double-check this and have a test that covers this.